mappings/net/minecraft/entity/passive/ChickenEntity.mapping

```diff
@@ -16,6 +17,10 @@ CLASS amc net/minecraft/entity/passive/ChickenEntity
 	METHOD ca initAttributes ()V
 	METHOD cv getDeathSound ()Lxm;
 	METHOD d getCurrentExperience (Larb;)I
+	METHOD dz isJockey ()Z
 	METHOD e getHurtSound (Lage;)Lxm;
+	METHOD f isBreedingItem (Lawo;)Z
 	METHOD k updateMovement ()V
 	METHOD n prepareGoals ()V
+	METHOD s setJockey (Z)V
+		ARG 1 isJockey
```

Comment:
If this is referring to the "Chicken Jockey" phenomenon, I believe the one riding the chicken is the jockey, not the chicken.